### ISSUE TITLE
template doesn't work for sbt version 1.3.5

### ISSUE THREAD

**Issue by nikiforo** [typelevel/ce3.g8#58]
```
 % sbt -version
sbt version in this project: 1.3.5
```

If I ask sbt to create the template, it will fail to do so:
```
% sbt new typelevel/ce3.g8          
[info] Loading global plugins from /Users/anikiforov/.sbt/1.0/plugins
[info] Set current project to empty (in build file:/Users/anikiforov/Proj/empty/)
[info] Set current project to empty (in build file:/Users/anikiforov/Proj/empty/)
name [Cats Effect 3 Quick Start]: 
package [com.example]: 
scala3 [yes/NO]: 
testlib-use-munit-cats-effect-3 [YES/no]: 
testlib-use-cats-effect-testing-specs2 [yes/NO]: 

Exiting due to error in the template: /var/folders/23/qdj88f9132b3z13xwz_t21400000gq/T/giter8-68711820940554
File: /var/folders/23/qdj88f9132b3z13xwz_t21400000gq/T/giter8-68711820940554/src/main/g8/build.sbt, 15:84: invalid character '-'
```
However, if I specify sbt version as in `build.properties` of this template
```
% sbt -Dsbt.version=1.5.5 new typelevel/ce3.g8
[info] welcome to sbt 1.5.5 (Oracle Corporation Java 13.0.1)
[info] loading global plugins from /Users/anikiforov/.sbt/1.0/plugins
[info] set current project to new (in build file:/private/var/folders/23/qdj88f9132b3z13xwz_t21400000gq/T/sbt_aff281ab/new/)
name [Cats Effect 3 Quick Start]: 
package [com.example]: 
scala3 [yes/NO]: 
testlib-use-munit-cats-effect-3 [YES/no]: 
testlib-use-cats-effect-testing-specs2 [yes/NO]: 
Skipping existing file: /Users/anikiforov/Proj/empty/empty/./cats-effect-3-quick-start/README.md
Skipping existing file: /Users/anikiforov/Proj/empty/empty/./cats-effect-3-quick-start/project/build.properties
Skipping existing file: /Users/anikiforov/Proj/empty/empty/./cats-effect-3-quick-start/.gitignore
Skipping ignored file: /var/folders/23/qdj88f9132b3z13xwz_t21400000gq/T/giter8-68943061636419/src/main/g8/src/test/scala/$package$/$if(testlib-use-cats-effect-testing-specs2.truthy)$.$endif$/HelloWorldSpec.scala

Template applied in /Users/anikiforov/Proj/empty/empty/./cats-effect-3-quick-start
```
Template will be created.

Akka mentions sbt requirement in it's instruction(https://github.com/akka/akka-quickstart-scala.g8) `sbt -Dsbt.version=1.3.6 new akka/akka-quickstart-scala.g8`. I think ce3.g8 should do the same.

**Comment by nikiforo** [comment]
In ce3 migration-guide  `sbt.version=1.5.0` is mentioned. Thus, `sbt -Dsbt.version=1.5.0 new typelevel/ce3.g8` ?

**Comment by kubukoz** [comment]
Makes sense :)

**Comment by kubukoz** [comment]
Fixed in #59
